src/main/java/com/unosquare/adminCore/controller/ContractController.java

```diff
@@ -0,0 +1,64 @@
+package com.unosquare.adminCore.controller;
+
+import com.unosquare.adminCore.entity.Contract;
+import com.unosquare.adminCore.service.ContractService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/contracts")
+public class ContractController {
+
+    @Autowired
+    ContractService contractService;
```

Comment:
I'll add a PR wide comment, but we really shouldn't be serializing data models for our request. Separate out into DTOs for requests to keep the payload to what you need and reduce the risk of lazy loading (here's a quick overview https://howtoprogramwithjava.com/hibernate-eager-vs-lazy-fetch-type/ )